README.md

```diff
@@ -545,6 +545,13 @@ or for all organisms with sufficient data:
 nomad job dispatch -meta ORGANISM= CREATE_COMPENDIA
 ```
 
+Compendia jobs run on the smasher instance.
+However they require a very large amount of RAM to be able to complete.
+Our smasher instance does not generally have enough RAM to be able to run them, so if you need to run a smasher job you should temporarily increase the size of the smasher instance.
+This can be done by changing the terraform variable `smasher_instance_type` which can be found in qinfrastructure/variables.tf`.
```

Comment:
I think you have a typo here

Reply:
I still think you want a second backtick at the start of infrastructure/variables.tf